src/dev/pages/file-picker/file-picker.ejs

```diff
@@ -1,34 +1,50 @@
-<h3 class="forge-typography--heading2">Default</h3>
+<forge-stack gap="24">
+ <div>
+  <h3 class="forge-typography--heading2">Default</h3>
+  <forge-file-picker id="file-picker-default" style="--forge-file-picker-width: 320px; width: 320px;">
+    <span slot="primary">Drag files here or</span>
+    <span slot="secondary">Secondary text here</span>
+    <forge-button variant="outlined">Select files</forge-button>
+    <span slot="helper-text" id="file-picker-helper-text"></span>
+  </forge-file-picker>
+  <ul id="file-picker-list"></ul>
+ </div>
 
-<forge-file-picker id="file-picker-default" style="--forge-file-picker-width: 320px; width: 320px;">
-  <span slot="primary">Drag files here or</span>
-  <span slot="secondary">Secondary text here</span>
-  <forge-button variant="outlined">Select files</forge-button>
-  <span slot="helper-text" id="file-picker-helper-text"></span>
-</forge-file-picker>
-<ul id="file-picker-list"></ul>
+ <div>
+  <h3 class="forge-typography--heading2">Compact</h3>
+  <forge-file-picker id="file-picker-compact" compact>
+    <forge-button variant="outlined">Select files</forge-button>
+  </forge-file-picker>
+  <ul id="file-picker-list-compact"></ul>
+ </div>
 
-<h3 class="forge-typography--heading2">Compact</h3>
 
-<forge-file-picker id="file-picker-compact" compact>
-  <forge-button variant="outlined">Select files</forge-button>
-</forge-file-picker>
-<ul id="file-picker-list-compact"></ul>
+ <div>
+  <h3 class="forge-typography--heading2">Borderless</h3>
+  <forge-card style="width: 400px; --forge-card-padding: 0;" raised>
+    <div style="height: 400px;">
+      <forge-file-picker id="file-picker-borderless"
+        style="--forge-file-picker-width: 100%; --forge-file-picker-height: 100%; --forge-file-picker-max-content-width: 300px;"
+        borderless>
+        <img slot="primary" src="https://cdn.forge.tylertech.com/v1/images/spot/cloud-storage-spot.svg" alt="" height="132px" />
+        <span slot="secondary">Drag files here or click the button above to manually select files</span>
+        <forge-button variant="outlined">Select files</forge-button>
+      </forge-file-picker>
+      <forge-ripple></forge-ripple>
```

Comment:
Unrelated, but can we replace usages of `<forge-ripple>` with `<forge-state-layer>` and `<forge-focus-indicator>` instead? We need to sweep this anyway so taking care of it as we go will help.